SimCalorimetry/HcalSimProducers/python/hcalUnsuppressedDigis_cfi.py

```diff
@@ -50,6 +52,9 @@
 from Configuration.StandardSequences.Eras import eras
 if eras.fastSim.isChosen():
     hcalSimBlock.hitsProducer = cms.string('famosSimHits')
+if eras.run2_HF_2016.isChosen():
```

Comment:
I realize that isChosen is not ideal, but the problem is that the hcalSimBlock PSet is imported by multiple configs. If I use toModify, the changes will not get picked up. You can see that FastSim also has to do this...